indico/modules/events/templates/display/indico/meeting.html

```diff
@@ -5,9 +5,13 @@
 
 {{ render_event_header_msg(event) }}
 
+{% set has_subheader = event.description or files or folders or lectures or conf.getOrgText()
```

Comment:
There are 2 problems: Most importantly, I cannot use the `:empty` selector to put rounded corners to the previous element. Secondly, the `.event-sub-header` is a `<table>` and when it's empty it's the `<tbody>` that is actually empty.